parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -457,3 +434,16 @@ def _send_event(self, event, **kwargs):
 
             # Return the entry id of the open file and the file descriptor
             return manifest.entry_id, self.local_storage.create_file_descriptor(manifest.entry_id)
+
+    async def file_resize(self, path: FsPath) -> EntryID:
+        # Check write rights
+        self._check_write_rights(path)
+
+        # Lock manifest
+        async with self._lock_manifest_from_path(path) as manifest:
+
+            # Perform resize
+            self._file_resize(manifest)
+
+            # Return entry id
```

Comment:
\# Challenge the usefulness of those comments
C'est vraiment bien utile ces commentaires ? ça m'a l'air bien redondant...

Reply:
Clairement pas utile. Apres ca saute aux yeux parce que cette transaction est vraiment simple, mais j'ai utilisé ce formalisme la pour toutes les transactions et ca m'aide pas mal à la lecture.